src/target/riscv32.c

```diff
@@ -67,8 +67,10 @@ typedef struct riscv32_regs {
 #define RV32_MATCH_BEFORE 0x00000000U
 #define RV32_MATCH_AFTER  0x00040000U
 
+static ssize_t riscv32_reg_read(target_s *target, const int c, void *data, const size_t max);
 static void riscv32_regs_read(target_s *target, void *data);
 static void riscv32_regs_write(target_s *target, const void *data);
+static ssize_t riscv32_reg_write(target_s *const target, const int c, const void *data, const size_t max);
```

Comment:
Please move this declaration up to just under the `riscv32_reg_read()` one for consistency.